mindsdb/integrations/handlers/hubspot_handler/hubspot_handler.py

```diff
@@ -15,28 +16,28 @@
 
 
 class HubspotHandler(APIHandler):
```

Comment:
To work with data catalog, handler's class should be inherited from [MetaApiHandler](https://github.com/mindsdb/mindsdb/blob/main/mindsdb/integrations/libs/api_handler.py#L513)

Reply:
Done